### ISSUE TITLE
Clicking scrollbar removes selection

### ISSUE THREAD

**Issue by plufz** [ThibaultJanBeyer/DragSelect#36]
**Describe the bug**
Just first wanted to say that I have made a hack to circumvent this bug for me, so I'm fine. I just wanted to report this, so you know of it. Close it if you don't want to fix it.

But clicking the scrollbar when you have a selection removes the selection.

**To Reproduce**
1. Go to https://codepen.io/anon/pen/qJdgNg
2. Make a selection
3. Click and drag the scrollbar
4. The selection is gone

**Expected behavior**
Items should still be selected.

**Desktop (please complete the following information):**
 - macOS
 - Chrome, Safari and Firefox

**Comment by ThibaultJanBeyer** [comment]
@13Nunes nope. That is the normal behavior for a selection. If you select something on your operating system, then click somewhere else, the selection is lost. If you had read the readme, you would have seen that there is a `multiSelectMode` property to change the behavior.

What @plufz is reporting is something different. He’s stating that the selection get’s lost when clicking on the scrollbar. Which is because the browser also sends a click event when clicking the scrollbar. Maybe there is a clean solution to detect whether it is a scrollbar click or not. I’ll look into plufz request but it’s not the highest priority

**Comment by 13Nunes** [comment]
It makes sense. Thanks for you answer.
I have implemented something different [here](http://www.be-br.com/institutounibanco/#/perception-game-stage-1) because this UX is better to mobile phones.
I recommend it with a new feature.

**Comment by jlindberg-oss** [comment]
@ThibaultJanBeyer great library, thank you for making.

I also noticed that even when you click on a button inside the area, you lose the selection.
This is true even if you do `event.stopPropagation()` inside the button's click handler.

After reading the code I added the following workaround with works for me:
```javascript
document.getElementById('myButton').addEventListener('mousedown', function(event) {
  event.stopPropagation();
});
```

Is there any other setting that I missed or is that the only way?  If not I think it's a bug, being able to select elements and then click on a button without losing the selection is pretty important.

**Comment by ThibaultJanBeyer** [comment]
Hey @jlindberg-oss, that is expected behaviour. You’ll have to either use the workaround you did, or create your own Class-mixing and override the `startUp` method to your likings or to place your buttons that should not clear the selection *outside of the area*.

**Comment by ThibaultJanBeyer** [comment]
Just fixed the scrollbar bug in version `1.12.2`
